src/Components/Aspire.Azure.Search.Documents/ConfigurationSchema.json

```diff
@@ -0,0 +1,132 @@
+                            },
+                            "LoggedContentSizeLimit": {
+                              "type": "integer",
+                              "description": "Gets or sets value indicating maximum size of content to log in bytes. Defaults to 4096."
+                            }
+                          },
+                          "description": "Gets the client diagnostic options."
+                        },
+                        "Retry": {
+                          "type": "object",
+                          "properties": {
+                            "Delay": {
+                              "type": "string",
+                              "format": "duration",
+                              "description": "The delay between retry attempts for a fixed approach or the delay\non which to base calculations for a backoff-based approach.\nIf the service provides a Retry-After response header, the next retry will be delayed by the duration specified by the header value."
+                            },
+                            "MaxDelay": {
+                              "type": "string",
+                              "format": "duration",
+                              "description": "The maximum permissible delay between retry attempts when the service does not provide a Retry-After response header.\nIf the service provides a Retry-After response header, the next retry will be delayed by the duration specified by the header value."
+                            },
+                            "MaxRetries": {
+                              "type": "integer",
+                              "description": "The maximum number of retry attempts before giving up."
+                            },
+                            "Mode": {
+                              "enum": [
+                                "Fixed",
+                                "Exponential"
+                              ],
+                              "description": "The approach to use for calculating retry delays."
+                            },
+                            "NetworkTimeout": {
+                              "type": "string",
+                              "format": "duration",
+                              "description": "The timeout applied to an individual network operations."
+                            }
+                          },
+                          "description": "Gets the client retry options."
+                        }
+                      },
+                      "description": "Provides the client configuration options for connecting to Azure\nCognitive Search."
```

Comment:
When we update to the new NuGet package version this will get updated in the same PR. We have build validation that ensures the generated json matches what is checked in.